ui/components/MesheryPerformance/PerformanceProfiles.js

```diff
@@ -215,32 +245,20 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar })
                 onClick={() => setProfileForModal({})}
               >
                 <AddIcon />
-              Add Performance Profile
+                Add Performance Profile
               </Button>
             </div>
           </Paper>
         ) : (
-          <div style={{ width : "fit-content", margin : "4rem auto 0" }}>
-            <Button
-              aria-label="Add Performance Profile"
-              variant="contained"
-              color="primary"
-              size="large"
-              // @ts-ignore
-              onClick={() => setProfileForModal({})}
-            >
-              <AddIcon />
-            Add Performance Profile
-            </Button>
-          </div>
+          console.log("yay")
         )}
 
         <GenericModal
           open={!!profileForModal}
           Content={
             <Paper style={{ margin : "auto", maxWidth : "90%", outline : "none" }}>
               <MesheryPerformanceComponent
-              // @ts-ignore
+                // @ts-ignore

```

Comment:
Was it needed?

Reply:
do:
```
cd ui
npx eslint . --fix
```

Reply:
Yes

Reply:
`condition==true && reactEle`   you can do it when you have to show something when the condition yields true, and have nothing to show when the condition is false.
running eslint will solve the lining errors and warning if any.